maat-court-data-api/src/main/java/gov/uk/courtdata/billing/repository/ApplicantCclfResetRepository.java

```diff
@@ -0,0 +1,28 @@
+package gov.uk.courtdata.billing.repository;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Collections;
+
+@Repository
+@RequiredArgsConstructor
+public class ApplicantCclfResetRepository {
+
+    private static final String RESET_CCLF_SQL = """
+        UPDATE APPLICANTS
+        SET SEND_TO_CCLF = NULL,
+            DATE_MODIFIED = SYSDATE,
+            USER_MODIFIED = USER
```

Comment:
For the other endpoints we have gone with passing in the username. The ask here hasn't been particularly clear but it feels like it makes sense - suggest maybe going with that as well for now, if we decide to change in the future then all the endpoints can be changed at once.